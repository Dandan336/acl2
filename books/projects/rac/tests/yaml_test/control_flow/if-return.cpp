// RAC begin

int foo(int a)
{
  if (a)
    return 1;

  return 0;
}

// RAC end
