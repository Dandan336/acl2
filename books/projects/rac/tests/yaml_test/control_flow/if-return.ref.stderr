if-return.cpp:8: syntax error, unexpected RETURN, expecting ELSE
